lib/load-migrations.js

```diff
@@ -16,8 +53,13 @@ function loadMigrationsIntoSet (options, fn) {
   const set = opts.set
   const store = opts.store
   const ignoreMissing = !!opts.ignoreMissing
+  const migrations = opts.migrations
   const migrationsDirectory = path.resolve(opts.migrationsDirectory || 'migrations')
+
+  // filterFn :: String -> boolean
   const filterFn = opts.filterFunction || (() => true)
```

Comment:
Can we remove these unrelated changes?

Reply:
I added these comments because it's hard to tell that the signatures of the `sortFunction` and `filterFunction` are different. `sortFunction` accepts a `Migration` as an argument but the `filterFunction` takes a `String` title of the migration. Perhaps they are better off in the `Readme` file. I will remove these...

Reply:
I would love it if sonmeone wanted to add some JSDoc to these. Would be nice to generate some types instead of having to maintain those manually.